packages/documentation/src/pages/architecture/front-end-standards/styles.mdx

```diff
@@ -0,0 +1,100 @@
+# Guidelines for good SCSS / CSS
```

Comment:
Cut out several of the implementation detail sections and left only the things related to standards or recommendations